arch/cortex-m/src/mpu.rs

```diff
@@ -132,6 +132,10 @@ register_bitfields![u32,
 const MPU_BASE_ADDRESS: StaticRef<MpuRegisters> =
     unsafe { StaticRef::new(0xE000ED90 as *const MpuRegisters) };
 
+/// Non-Secure alias for the MPU.
+const MPU_NS_BASE_ADDRESS: StaticRef<MpuRegisters> =
+    unsafe { StaticRef::new(0xE002ED90 as *const MpuRegisters) };
+
```

Comment:
I'm not the expert on this, but I think we are going to need to re-factor how the MPU is organized in these tock crates so that we don't have the `unsafe StaticRef` for an address we don't actually want to map these registers to.

We probably need to do something like what we do with chips were we separate the MPU implementation into a different from from the register instantiation.

Reply:
@JADarius  See #4427 for an alternative now (well, once that's merged)